dfm-sideline-sidekick-app/pages/GlobalSearch.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { FlatList, Text, TextInput, View } from "react-native";
+import Icon from "react-native-vector-icons/FontAwesome"; 
```

Comment:
This import gave me an error, but looks like the native vector icons package should be added in the general principles pr, so should be ok